src/components/InfiniteHits.vue

```diff
@@ -25,6 +25,7 @@
       :refine-previous="refinePrevious"
       :refine-next="refineNext"
       :refine="refineNext"
+      :insights="state.insights"
```

Comment:
We could also add the function to the `item` slot. It would avoid to recreate the full UI to use insights.

Reply:
Yes we can!
At first I thought maybe it would confuse some users as on why you have to pass objectIDs on the item slot scope, but that's probably unfounded.

```ts
<ais-infinite-hits>
  <div slot-scope="{ item, refine, insights }">
      <button @click="insights('clickedObjectIDsAfterSearch', { eventName: 'Add to cart', objectIDs: [item.objectID] })">Add to cart</button>
  </div>
</ais-infinite-hits>
```

Reply:
Ahh I get the point, I don't think we matter too much in that case. The signature remains the same between the slots that's what is important. Worst case if we have users complain we could pre-apply the function with the `objectID`.

Reply:
added here: b03c9c032ed1af5375d164b5e957bfec90913690